src/groundlight/unstableapi.py

```diff
@@ -0,0 +1,79 @@
+"""
+unstableapi.py
+
+This module is part of our evolving SDK. While these functions are designed to provide valuable functionality to enhance your projects, it's important to note that they are considered unstable. This means they may undergo significant modifications or potentially be removed in future releases, which could lead to breaking changes in your applications.
+"""
+import json
+from typing import Union
+
+from model import (
+    Channel,
+    Detector,
+    Rule,
+    RuleCreationInput,
+    SnoozeTimeUnit,
+    Verb,
+)
+from openapi_client.model.action import Action
+from openapi_client.model.condition import Condition
+from openapi_client.model.rule_creation_input import RuleCreationInput
+
+from groundlight import Groundlight
+
+
+class UnstableApi(Groundlight):
```

Comment:
Written like this, you can call
`from groundlight import UnstableApi
gl = UnstableApi`
and gl will behave as if it was the Groundlight client but with this extra feature. The previous commit had this written into the Groundlight class but we want to make this readily apparent that this feature is unstable